modules/statics/src/base.ts

```diff
@@ -124,6 +124,10 @@ export enum CoinFeature {
    * Are fees for transactions of this coin paid for by the Enterprise (eg, Enterprise gas tank)?
    */
   ENTERPRISE_PAYS_FEES = 'enterprise-pays-fees',
+  /*
+   * This coin requires a reserve amount to fund a wallet
```

Comment:
nit: The reserve is not just for funding
```suggestion
   * This coin requires that accounts keep a minimum balance as reserve
```